SPEC.md

```diff
@@ -182,6 +183,33 @@ Contents give more information about the issue's check, including a description
 
 Some engines require the ability to refer to other source locations. For this reason, the Issue type has an optional `other_locations` field, which is an array of other `Location` items that this issue needs to refer to.
 
+## Versioning
+
+This specification is versioned. The current version is [in the repository](https://github.com/codeclimate/spec/blob/master/VERSION). Engines declare the version of the specification they are compatible with in the manifest file, described below.
+
+## Manifest File
+
+All engines must include a `manifest.json` file at `/manifest.json`. This file, known as the manifest, includes information that is necessary for the analysis runtime. Here is an example of a manifest:
+
+```
+{
+  "name": "govet",
+  "maintainer": {
+    "name": "Michael R. Bernstein",
+    "email": "mrb@codeclimate.com",
+  },
+  "spec_version": "0.0.1",
+}
+```
+
+The fields in the manifest include:
+
+* `name` (`String`) - the name of the package.
+* `maintainer` (`Object`)
+  * `name` (`String`) - the name of the maintainer
+  * `email` (`String`) - the email address of the maintainer
+* `spec_version` (`String`) - the most recent version of the specification which this engine supports
```

Comment:
If we're intending to follow semantic versioning for the spec (which I'm in favor of), then I might suggest the following:

Keep the format as what you have, a single field with a single version value.

Interpret that value like the `~>` operator in gemspecs.

So:

```
spec_version: 0.0.1 means
  I (claim to) support >= 0.0.1 && < 0.1.0

spec_version: 0.1 means
  I support >= 0.1.0 && < 1.0.0

spec_version: 1 means
  I support everything (>= 1.0.0 && < *)
```

(I think I have that right, correct me if I'm wrong)

WDYT?